vendor/CEFWebView/fix_cef_framework.sh

```
@@ -0,0 +1,34 @@
+    exit 0
+fi
+
+echo "🔧 Fixing CEF framework symlinks at: $FRAMEWORK_PATH"
+
+# Remove mis-embedded root files (should be symlinks, not copies)
+rm -f "$FRAMEWORK_PATH/Chromium Embedded Framework"
+rm -f "$FRAMEWORK_PATH/Resources"
+rm -f "$FRAMEWORK_PATH/Libraries"
+
+# Create proper root-level symlinks to Versions/Current
+cd "$FRAMEWORK_PATH"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)